components/button/floating-buttons.js

```diff
@@ -0,0 +1,261 @@
+import 'lit-media-query/lit-media-query.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { buttonStyles } from './button-styles.js';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { styleMap } from 'lit-html/directives/style-map.js';
+
+class FloatingButtons extends RtlMixin(LitElement) {
+	static get properties() {
+		return {
+			/**
+			 * Whether always enable floating buttons. Note: by default, buttons may not float depending on
+			 * space available since floating buttons may result in poor user experience when view-port is small
+			 * (ex. phones).
+			 */
+			alwaysFloat: {
```

Comment:
In contrast to Polymer, mixed-case properties in Lit do not automatically map to lower-case + hyphen attributes.  So to preserve the API for these, I think we will need:
```
alwaysFloat: { type: Boolean, attribute: 'always-float' },
```
We should be able to test this out by adding the `always-float` attribute in the demo.  Note: looks like legacy ActionButtons use this attribute via their `UseFloatingButtons`, which is exposed through the legacy `PageControl`, and set in at least one place... email.   :(